src/index.html

```diff
@@ -10,8 +10,188 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body> 
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="inputs-block">
+        <legend> 
+          Personal information
+        </legend>
+        <label class="input-row">
+        Surname:
+          <input 
+          type="text" 
+          name="sname"
+          id="sname" 
+          autocomplete="off" 
+          required>
+        </label><br>
+        <label class="input-row">
+          Name:
+          <input 
+          type="text" 
+          name="lname" 
+          id="lname"
+          autocomplete="off"
+          required
+          >
+        </label><br>
```

Comment:
I think you don't need br after each label, it creates an extra space after each input so that your work differs a lot from the reference picture

```suggestion
        </label>
```

Reply:
not fixed everywhere